modules/service/src/main/scala/ch/epfl/bluebrain/nexus/commons/service/retryer/TaskRetry.scala

```diff
@@ -6,7 +6,7 @@ import monix.execution.CancelableFuture
 import monix.execution.Scheduler.Implicits.global
```

Comment:
the scheduler should rely on the akka dispatcher (construct an implicit value based on an implicit execution context available to the retry function)